fastapi_websocket_rpc/utils.py

```diff
@@ -52,9 +57,28 @@ def random_datetime(start=None, end=None):
 class StringUtils(object):
     @staticmethod
     def convert_camelcase_to_underscore(name, lower=True):
-        s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
-        res = re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1)
+        s1 = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", name)
+        res = re.sub("([a-z0-9])([A-Z])", r"\1_\2", s1)
         if lower:
             return res.lower()
         else:
             return res.upper()
+
+
+# Helper methods for supporting Pydantic v1 and v2
+def is_pydantic_pre_v2():
+    return version.parse(pydantic.VERSION) < version.parse("2.0.0")
+
+
+def get_model_serializer():
+    if is_pydantic_pre_v2():
+        return lambda model, **kwargs: model.json(**kwargs)
+    else:
+        return lambda model, **kwargs: model.model_dump_json(**kwargs)
+
+
+def get_model_parser():
```

Comment:
Wouldn't it be clearer to have methods `pydantic_parse(model, **kwargs)` and `pydantic_serliaze(model, **kwargs)`.
So one call would be enough instead of first retrieving the correct function and only then using it...

Reply:
Agreed. ChatGPT helped me code that :smile: doing it in one call is better